### ISSUE TITLE
Use sockets in the agent/server connection to deliver streams rather than chunked data.

### ISSUE THREAD

**Issue by kjlape** [thebentern/monitor#3]
Pub/Sub in any form is starting to show some weaknesses for this use case. Maybe we can switch our implementation to a persistent, streaming connection (e.g. socket) in the agent.

**Comment by kjlape** [comment]
@thebentern What are your thoughts? I'm looking into the tech requirements...

**Comment by thebentern** [comment]
I agree about Pub/Sub showing its weakness in our core functionality. This was my idea for working around the persistence issue, which would also buy us accuracy in terms of time-stamping messages. 
https://github.com/thebentern/monitor/issues/1
What do you think?
